couchdbkit/version.py

```diff
@@ -4,6 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-from six.moves import map
-version_info = (0, 9, 1)
-__version__ =  ".".join(map(str, version_info))
+version_info = (0, 9, 2)
+__version__ = ".".join([str(vi) for vi in version_info])
```

Comment:
Recommend setting your dev environment to include a newline at the end of file by default

Reply:
good idea, Ill do this